### ISSUE TITLE
Add switch to simple text view instead of UI

### ISSUE THREAD

**Issue by Chris2011** [javierllorente/netbeans-rest-client#18]
I really like this plugin and the UI, you made it really great and it is similar to postman, so thanks for this. Also I like the idea of what VS Code is doing with the simple http rest client, where you just open a new *.http file and you have syntax highlighting and code completion for HTTP stuff like METHODS, HEADERS, etc. So I would like to add this to so that it is synced with the UI or just switch between the UI. Atm my prio is just to add a lexer/parser for http files and switch between source and preview like a multiviewelement what you probably know from swing components in NetBeans. You can assign this ticket to me.

**Comment by javierllorente** [comment]
So it would be like a raw mode in which you can compose (or open) a json that defines your request (method, body, etc.), right?

**Comment by Chris2011** [comment]
Yes, but it will be not a json it will be an own language with those HTTP commands, etc. See the vs code rest client here: https://marketplace.visualstudio.com/items?itemName=humao.rest-client

**Comment by javierllorente** [comment]
I've already seen that. To be more precise, JSON-based language :-)
Perhaps there could be two buttons under "Send": "Normal" and ".http" or something similar.
As you know, if you need feedback, let me know. Thanks!

**Comment by Chris2011** [comment]
yeah, smth like that :)

**Comment by Chris2011** [comment]
I just need to handle the ANTLR lexer and parser and everything else. Maybe step by step. First syntax coloring, execution of the request and next code completion

**Comment by Chris2011** [comment]
Update: After long run, I'm having a simple colored HTTP file, veeeeeery simple and not full filled but we can make a comment for the first run: "For full feature, please use the GUI instead of the text file".

**Comment by javierllorente** [comment]
It is a beginning :)
First run message only for new users (on install)?
I would like to try it.

**Comment by Chris2011** [comment]
No, it is just a comment when you open the .http file in head that they should use the GUI instead of this. I'm still working on it. But soon it is coming :)

**Comment by Chris2011** [comment]
So finally. I added a todo list and I will create a PR soon. Unfortunately it will be a big one.
What I have done so far:
Lexer and parser with ANTLR, syntax highlighting, semantic highlighting for HTTP keyword, showing errors, just from ANTLR, need more improvement, simple but sometiems broken code completion. For the other stuff I will create more tickets and smaller PRs I promise :D. Also if you think it is better, I can try to minimize the PR overhead and make more smaller ones instead of the big one.

**Comment by Chris2011** [comment]
I thought that this is handled by the response panel itself. So if you handle this there, it should  work. So maybe the exception is not proper handled. Will check this. If you can do me a favor, you can also check how you handle this exception. Maybe you can add this directly somehow to the repsonsePanel. If not, I will check your code, so that I can handle it by my own.

**Comment by javierllorente** [comment]
I have centralised it in ExceptionUtils.handleAndDisplayProcessingException()
Commit 764ef36
